gitpod-io/gitpod
# Upgrade Gradle for compatibility with Java 17

## Description

Currently, when we open Gitpod Workspace, it displays warnings about not being able to use the installed version of _Gradle_ with _Java 17_ and asking us to upgrade it to _v7.3.1_:

<img width="469" alt="image" src="https://user-images.githubusercontent.com/418083/183723797-b69e7f1f-e262-46fd-bceb-ea10d81bbfdf.png">

These notifications come from  [Language Support for Java](https://open-vsx.org/extension/redhat/java) extension from VS Code. And this PR contains the modifications generated by it.

## Related Issue(s)

None.

## How to test

1. Open the preview environment generated for this branch
2. Choose the stable version of IntelliJ IDEA as your preferred editor
3. Start a workspace using this repository: https://github.com/gitpod-io/spring-petclinic
4. Verify that the workspace starts successfully
5. Verify that the IDE opens successfully

<img width="1252" alt="image" src="https://user-images.githubusercontent.com/418083/183723255-dc6df30d-879f-486e-a917-9a7f526b294f.png">


## Release Notes

```release-note
NONE
```

## Documentation

None.

## Werft options:
